modules/farm_fd2/src/entrypoints/cover_crop_seeding/lib.submitOptional.unit.cy.js

```diff
@@ -0,0 +1,128 @@
+import { lib } from './lib.js';
+
+describe('Submission using the cover_crop lib.', () => {
+  // form with no winter kill
+  let formNoWinterKill = {
```

Comment:
Because each form is only used in one `it`, I think it makes sense to make each form object a local variable in the `it` that uses it..